pkg/api/v1/events/handlers.go

```diff
@@ -167,6 +167,9 @@ func (h *Handler) updateCondition(ctx context.Context, streamEvent *ptypes.Strea
 		}).Info("condition merge failed")
 
 		// XXX: should this be an nak, instead of a ack.
+		//  - (joel) I don't think so, since this could be an out of order message and
+		// .  MergeExisting returns an error because the state transition cannot be done.
+		//    This could be a NakTerm instead so we can have metrics on it?
```

Comment:
I agree a `Term` would be appropriate here. On a Nak (in a perfect world 😅 ) the message will be redelivered, but the redelivery couldn't succeed, and if it did, it would be an error.